integration_tests/tests/forseti/controls/logs-debug_level.rb

```diff
@@ -27,19 +27,14 @@
   end
   instance_id = JSON.parse(instance_describe_cmd.stdout)["id"]
 
-  # Set the log level to debug
-  describe command("forseti server log_level set debug") do
-    its('exit_status') { should eq 0 }
-    its('stdout') { should match /isSuccess": true/}
-  end
-
   # Note the timestamp
   timestamp = Time.now.getutc.strftime('%Y-%m-%dT%H:%M:%S.%3NZ')
 
-  # Create inventory
-  inventory_cmd = command("forseti inventory create --import_as #{random_string}")
+  # Set debug log level to debug and create inventory
+  inventory_cmd = command("forseti server log_level set debug && forseti inventory create --import_as #{random_string}")
```

Comment:
Fix: Seems like running the command to set log_level before inventory create does not "stick", this works.